image-specs/el7.3-base

```diff
@@ -0,0 +1,16 @@
+#base=libguestfs:centos-7.3
+#name=el7.3-base
+#osinfo=centos7.3
+#distro=el7
+#version=v1
+#arch=x86_64
+#expand=/dev/sda2
+#lvexpand=/dev/cl/root
+#root-partition=/dev/cl/root
+root-password password:123456
+update
+install qemu-guest-agent
+
+# fix for bz#1302678
```

Comment:
Could be that this bug was fixed.
I see that we have 3 places in which relabeling can be done:
1. Add it to the spec of the image.
2. Add it to 'virt-sysprep' during the image creation.
3. Don't do nothing and just let Lago do it (when it runs sysprep).

Which one should I use ?